Sources/JXKit/JXContext.swift

```diff
@@ -91,40 +93,103 @@ open class JXContext {
 }
 
 extension JXContext {
```

Comment:
I played around with eval() and I settled on this approach. We now have 3 eval() types: eval(), evalClosure(), evalModule(). And each has a variant for a String and a variant for a script resource to load.

The new evalClosure() variant interprets the code as a closure body, returning whatever the code 'return's. Within the code you can use $0, $1, etc for closure arguments, and as you'll see below the evalClosure() func has an optional withArguments: param. This replaces the previous 'withValues' that I added. That had to do all sorts of shenanigans with setting and unsetting $0, $1, etc globals. The closure model doesn't have to do that, plus it has the advantage of giving your code a local scope. So it can access and modify existing globals, but anything it declares is local.